pl_examples/pytorch_ecosystem/pytorch-forecasting/logo.svg

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
```

Comment:
Are we allowed to use their logo? Referencing to them is different to having their logo in our repo...

Reply:
It is MIT license and I am Pytorch Forecating’s creator. Absolutely fine with me.